.deepsource.toml

```diff
@@ -0,0 +1,14 @@
+version = 1
+
+test_patterns = ["**/**_test.py"]
+
+[[analyzers]]
+name = "python"
+enabled = true
+dependency_file_paths = [
+  "monkey/monkey_island/requirements.txt",
+  "monkey/infection_monkey/requirements.txt"
+  ]
+
+  [analyzers.meta]
+  runtime_version = "3.x.x"
```

Comment:
We don't need this file until we integrate deepsource. I think you should remove this file from this PR and instead make an issue called `Integration of deepsource` where you can put the contents of this file. Once we start deepsource integration, we'll pick it up.